src/stores/GuildChannelStore.js

```diff
@@ -22,6 +22,15 @@ class GuildChannelStore extends DataStore {
     return channel;
   }
 
+  /**
+     * The parent {@link Channel} of the {@link GuildChannel}s in this store *(Only if the store
+     * was retrieved from {@link CategoryChannel}.children)*.
+     * @type {?GuildChannel}
+     * @readonly
+     */
+  get parent() {
```

Comment:
What's the reason for the getter of the private property? Can't it be just a property of the class (`this._parent = null` in the constructor), and then you skip a getter

Reply:
Someone *could* potentially pass the result of .children to somewhere without also passing in the parent category. Without the getter on the store, the only way to then get the parent would be to do `children.get(...).parent`.

Its pretty niche I'll grant you, but to me seems in keeping with the rest of the library (For example, GuildChannel's have a Guild property).

Reply:
Then make `parent` a public property, and remove the rather useless private parent property?

Reply:
Not had my coffee so apologies if I'm being a bit slow, but surely it needs to be read only, otherwise someone might thing they could do this to change the parent of a load of channels at once:
```js
<GuildChannel>.parent.children.parent = <CategoryChannel>;
```
I've just looked at what GuildChannel does, and that has a public `parentID` and a getter for `parent` - would that be more suitable do you think...?

Reply:
> otherwise someone might think they could do this to change the parent of a load of channels at once

This could still be the case with the guild's parentID, no? Nothing's stopping someone from going woopty doo `channel.parentID = "my ID";`.. So I think it would be fine? I'd wait for others opinions too tho